pkgs/applications/virtualization/bochs/default.nix

```diff
@@ -1,10 +1,49 @@
-{ stdenv, fetchurl, config
+{ stdenv, fetchurl
 , pkgconfig, libtool
 , gtk2, mesa, readline, libX11, libXpm
 , docbook_xml_dtd_45, docbook_xsl
 , sdlSupport ? true, SDL2 ? null
-, termSupport ? true , ncurses ? null
+, termSupport ? true, ncurses ? null
 , wxSupport ? true, wxGTK ? null
+# A plethora of configurable options
```

Comment:
Let's not do this, its already possible to just override `configureFlags` with `overrideAttrs`.